4term/nodejs/lab5/src/userDb.ts

```diff
@@ -0,0 +1,46 @@
+export interface User {
+    id: number;
+    username: string;
+    name?: string;
+}
+
+export class UserDatabase {
+    private users: User[] = [];
+
+    addUser(username: string, name?: string) {
+        const id = this.users.length;
```

Comment:
this is not ok. If you will delete one entry and then add new user you will get duplicated id's